src/generators/essudpgen/ReadoutGeneratorBase.h

```diff
@@ -30,7 +31,6 @@ class ReadoutGeneratorBase {
     uint64_t SpeedThrottle{0};       // 0 is fastest higher is slower
     uint64_t PktThrottle{0};         // 0 is fastest
     uint8_t headerVersion{1};        // v1 header by default
-    std::string FilePath{"none"};            // Record data file to read from
```

Comment:
Why not keep the filename in the base class?

Reply:
So this setting in maybe not relevant to all modules. Currently biforst using it only, so I moved this setting parameter to bifrost generator.